lib/storage/providers/__mocks__/SQLiteStorage.js

```diff
@@ -0,0 +1,3 @@
+import provider from '../AsyncStorage';
+
+export default provider;
```

Comment:
We decided to put the mocks on the `lib/storage` level - that's why other storage providers don't have mocks 

Whether we have a mocked storage or a storage that returns a mocked provider it's the same thing, but it's more convenient to have the mock on the upper level and write just one mock, rather than having to write a mock for each provider

Reply:
Are you sure that mock is in use? I think it's just not used at all and you use AsyncStorage mock. Correct me if I'm wrong :).

Reply:
Yeah, it's slightly more tangled than that - we do sometimes use the localforage mock 
Anyway I think it shouldn't be a concern for this PR.
Kudos for making the tests work with as little changes!